bin/drush-playwright-internal

```diff
@@ -39,7 +39,25 @@ if (!$test_id) {
   exit(1);
 }
 
-include_once '/var/www/html/web/core/includes/bootstrap.inc';
+// Get the docroot directory from the environment variable.
+$docroot_dir = getenv('DOCROOT');
```

Comment:
When we talked in Slack, I thought I mentioned instead doing this by parsing out the docroot from composer.json. I actually had to do this as well over in Drainpipe:

https://github.com/Lullabot/drainpipe/pull/613/files#diff-7adbb32c269b786500345b2dc08d60e8f89e3366137646bfce85cb1cbaf5f5fdR40-R45

Is there a reason this needs to be customizable outside of composer.json? I'm totally open to it if there is, I'd just like to understand why.